opengrok-indexer/src/main/java/org/opengrok/indexer/configuration/RuntimeEnvironment.java

```diff
@@ -1230,27 +1283,27 @@ public String getConfigURI() {
     }
```

Comment:
config uri doesn't need locking?

Reply:
I think this is not necessary. Firstly, it is not part of configuration, secondly it gets set only initially.